Source/Scene/Camera.js

```diff
@@ -2738,6 +2738,7 @@ define([
     Camera.clone = function(camera, result) {
         if (!defined(result)) {
             result = new Camera(camera._scene);
+            result.frustum = camera.frustum.clone();
```

Comment:
Actually I might just remove that line since I'm no longer cloning cameras that have orthographic frustums, unless this is something we want to support anyways.